dobi.yaml

```diff
@@ -2,17 +2,18 @@
 # alias
 # ===================================================
 
-alias=bb-buildingblock-build:
-  tasks: []
+alias=bb-gitversion-build:
+  tasks:
+  - gitversion-build
   annotations:
-    description: "-> build all content (not implemented yet)"
+    description: "-> build all content"
 
-alias=bb-buildingblock-deploy:
+alias=bb-gitversion-deploy:
   tasks: []
   annotations:
     description: "-> deploy all content (not implemented yet)"
 
-alias=bb-buildingblock-test: 
+alias=bb-gitversion-test: 
```

Comment:
deadbeef? it there something we test in this context?

Reply:
i dont want to delete the defaults. apart from this, actually i work on tests for gitversion.